packages/tokens/src/spacing.ts

```diff
@@ -15,6 +15,7 @@ const spacing = {
   1200: 48,
   1400: 56,
   1600: 64,
+  1800: 72,
```

Comment:
is this the right number... it's only 6 away from 1600 so i think it shoudl be 1750 not 1800

Reply:
This should be correct: 18 x 4 = 72

Reply:
ugh you're right

Reply:
@bruugey it's 8 away

1800 / 100 = 72 / 4 = 18